### PR TITLE
Dylansworkfordec21

### PR DESCRIPTION
**Pull request by tonyfruzza**
Here's the code change I want you to review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+## Requiments
+Python 3 required 
+
+no modules required
+## Usage
+This is a text based game, so all you need to do is type in the
+
+number that is behind your answer.
+## Contribution
+Commit directly from the main branch
```
Comment (tonyfruzza) [inline README.md]: Branch from `main` for your *feature branch*

File: README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+Make a pull request about your branch
+
+This code is for python and uses if statements, functions, inputs, print statements, and time to make the project
```
Comment (tonyfruzza) [inline README.md]: This is simply a duplication of your overview, why? I would add more about your PR instead, like add at least one reviewer. Also, you can add a note that is to say to search for open `issues` and refer to those in your PR

**Review by tonyfruzza — COMMENTED** [review]
Looks pretty good, but I left a couple comments for you to resolve

**Review by dylan-fruzza — APPROVED** [review]
cool

**Pull request merged**